dist/wrapper.js

```diff
@@ -2,12 +2,91 @@
 /* @flow */
 'use strict'
 
+/* ::
+import type {BmRequest, Headers} from './wrapper.js'
+*/
+
+const privateVars = new WeakMap()
+
+class BmResponse {
+  constructor (
+    request /* : BmRequest */
+  ) {
+    privateVars.set(this, {
+      headers: {},
+      payload: undefined,
+      request,
```

Comment:
It doesn't seem as though `request` is used at all. Consider dropping the argument, and not storing it in private state?